tests/acceptance/home/user/user-test.js

```diff
@@ -0,0 +1,87 @@
+import { module, test } from 'qunit';
+import { visit, currentURL, click } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import moment from 'moment';
+
+module('Acceptance | user', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(async function () {
+    this.sessionService = this.owner.lookup('service:session');
+    const store = await this.owner.lookup('service:store');
+
+    const loggedUser = await this.server.create('user');
+    const loggedUserModel = await store.findRecord('user', loggedUser.id);
+
+    this.set('store', store);
+    this.set('sessionService.currentUser', loggedUserModel);
+
+    const observation1 = {
+      id: 1,
+      birdname: 'drozd',
+      latLocation: 49.664354,
+      lngLocation: 19.184909,
+      observationDate: moment('2022-06-10').toDate(),
+      notes: 'some informations that are important to user/others.',
+      owner: this.sessionService.currentUser,
```

Comment:
In our cases we had to declare relations using id like: 
```js
owner.id=this.sessionService.currentUser.id
```
but I think it's ok if it works